package.json

```diff
@@ -19,6 +19,7 @@
   },
   "homepage": "https://github.com/behance/lightbox#readme",
   "dependencies": {
+    "idle-timer": "github:nemtsov/idle-timer#6061205",
```

Comment:
Will convert this to a version once https://github.com/micnews/idle-timer/pull/3 is merged and a new version is released.